free-programming-books.md

```diff
@@ -1850,13 +1852,17 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
 
 #### Pandas
 
-* [Learn Pandas](https://bitbucket.org/hrojas/learn-pandas) - Hernan Rojas (0.13.0)
+* [Learn Pandas](https://bitbucket.org/hrojas/learn-pandas) - Hernan Rojas (0.18.1)
 
 
 #### Pyramid
 
 * [Pyramid for humans](http://docs.pylonsproject.org/projects/pyramid-tutorials/en/latest/humans/index.html)
 
+#### Tornado
+
+* [Learn Web Programming](https://bitbucket.org/hrojas/learn-web-programming)
+
 
```

Comment:
I tried to remove any extra spaces but I didn't see any. Maybe I missing something real obvious. I tried to commit again.